meta-resin-common/recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -0,0 +1,42 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/balena-files:"
```

Comment:
nit.. os-files instead of balena-files?

Reply:
I'll leave it for now as it is. Thanks for the suggestion.